plugin/pulpcore/plugin/stages/association_stages.py

```diff
@@ -0,0 +1,158 @@
+import asyncio
+from collections import defaultdict
+
+from django.db.models import Q
+
+from pulpcore.plugin.models import ProgressBar
+
+
+class content_unit_association:
```

Comment:
Class name should be CapWords - PEP8.

Reply:
Same question as [here](https://github.com/pulp/pulp/pull/3559#discussion_r206939231).